stepbystep/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>WebAssembly</title>
+  <link href="../css/primer.css" rel="stylesheet">
+  <link href="../css/custom.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <div class="columns">
+      <div class="two-thirds column centered">
+        <section class="header text-center">
+          <h1 class="title">WebAssembly</h1>
+          <a class="btn btn-outline" href="../">Overview</a>
+          <a class="btn btn-outline" href="../demo/">Demo</a>
+          <a class="btn btn-outline" href="https://github.com/WebAssembly/design">Design</a>
+          <a class="btn btn-outline" href="https://github.com/WebAssembly/spec">Specification</a>
+          <a class="btn btn-outline" href="https://www.w3.org/community/webassembly/">Community Group</a>
+          <a class="btn btn-outline" href="./">Step by Step</a>
+        </section>
+      </div>
+    </div>
+    <div class="columns">
+      <div class="three-fourths column centered">
+        <h2>Step by Step Setup</h2>
+        <p>
+          This page aims to provide complete, working step-by-step instructions
+          to compile a simple program directly to WebAssembly.
+        <p>
+        <h3>Install the correct version of cmake</h3>
+        <p>
+          To compile the rest of the tools, cmake is required. Specifically,
+          LLVM requires cmake 3.4.3 or higher. If you already have cmake 3.4.3
+          or higher, or are able to install or upgrade cmake through your
+          package manager, you can skip this section.
+        </p>
+        <p>
+          The latest version of cmake comes precompiled and can be downloaded
+          with the following commands:
+        </p>
+        <pre>
+          $ curl https://cmake.org/files/v3.6/cmake-3.6.0-Linux-x86_64.tar.gz \
+                 -o cmake-3.6.0-Linux-x86_64.tar.gz
+          $ gunzip cmake-3.6.0-Linux-x86_64.tar.gz
+          $ tar -xf cmake-3.6.0-Linux-x86_64.tar
+          $ PATH=`pwd`/cmake-3.6.0-Linux-x86_64/bin:$PATH
+        </pre>
+        <h3>Install the correct version of LLVM</h3>
+        <p>
+          To compile to WebAssembly, emscripten requires a "vanilla" build of LLVM
```

Comment:
A vanilla build is one option of 2, the other is to use emscripten's LLVM and asm2wasm. That's currently the [recommended route](https://github.com/kripken/emscripten/wiki/WebAssembly) as the wasm backend is not stable yet.

I think changing to that would be a simplification here - there is no need to get and build LLVM by itself (the emsdk will do that), and there is no need for the `EMCC_WASM_BACKEND` env var.